packaging/epel/convert2rhel.spec

```diff
@@ -1,6 +1,12 @@
+%if 0%{?epel} && 0%{?epel} <= 7
```

Comment:
Why are you using `%epel`? There are very few circumstances where using `%epel` makes sense over `%rhel`.

Reply:
Hi @Conan-Kudo, the reason is the following paragraph in [the Fedora Packaging Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_spec_legibility):
> To help facilitate legibility, only macros and conditionals for Fedora and EPEL are allowed to be used in Fedora Packages. Use of macros and conditionals for other distributions, including Fedora derivatives, is not permitted in spec files of packages in the main Fedora repositories unless those macros and conditionals are also present in Fedora.

If it's not that hot with this rule, I can definitely change it to `%rhel` (and maybe even merge the two spec files into one).

Reply:
`%rhel` is considered an EPEL conditional. All the macros exposed by RHEL and CentOS are permitted.

Reply:
Ok, I'll change this. Thanks for pointing that out.